src/main/webapp/app/components/util/secured-image.component.ts

```diff
@@ -35,9 +42,29 @@ export class SecuredImageComponent implements OnChanges {
     // we need HttpClient to load the image and DomSanitizer to trust the url
     constructor(private httpClient: HttpClient, private domSanitizer: DomSanitizer) {}
 
+    // triggers the reload of the picture when the user clicks on a button
+    retryLoadImage() {
+        this.retryCounter = 0;
+        this.endLoadingProcess.emit('loading');
+        this.ngOnChanges();
+    }
+
     private loadImage(url: string): Observable<any> {
-        return this.httpClient
+        const element = this.httpClient
```

Comment:
Why?

Reply:
we can also make a return without using an additional attribute